http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/ReactiveRestClientIT.java

```diff
@@ -255,6 +257,16 @@ public void malformedChunk() {
         Assertions.assertEquals("io.vertx.core.http.HttpClosedException", response.body().asString());
     }
 
+    @Test
+    @EnabledOnNative
+    // reproduced for: https://github.com/quarkusio/quarkus/issues/36986
```

Comment:
We usually use `@Tag` for this (eg https://github.com/quarkus-qe/quarkus-test-suite/blob/main/http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/DomainSocketIT.java#L29)

Reply:
changed